gemfiles/rails_61/Gemfile

```diff
@@ -13,6 +13,7 @@ end
 group :rubocop do
   gem 'rubocop'
   gem 'rubocop-packaging'
+  gem 'rubocop-performance'
 end
```

Comment:
We shouldn't be including rubocop for the `gemfiles` because we only run Rubocop at the root in CI. These are never used. We've made that change in ActiveAdmin. Would you like to do that removal here or in a separate PR? Either is fine with me.